charts/goblackhole/templates/bundle.yaml

```
@@ -0,0 +1,45 @@
+{{/*
+
+Copyright © 2021 Bedag Informatik AG
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/}}
+{{- include "bedag-lib.manifest.bundle" $ | nindent 0 }}
+
+{{/*
+  Define Goblackhole Chart Bundle
+*/}}
+{{- define "chart.bundle" -}}
+resources:
+  - type: "deployment"
+    values: {{ toYaml $.Values.deployment | nindent 6 }}
+    overwrites:
+      volumeMounts:
+        - name: "goblackhole-config"
+          mountPath: "/etc/goblackhole/config.yaml"
+          subPath: config.yaml
+      volumes:
+        - name: goblackhole-config
+          configMap:
+            name: goblackhole-config
```

Note that the user no longer has the ability to add his own `volumeMounts`& `volumes`. Since you òverwrites` implements `mergeOverwrite`functionality

```suggestion
      volumeMounts:
        {{- with $.Values.deployment.volumeMounts }}
          {{- toYaml . | nindent 8 }}
        {{- end }}
        - name: "goblackhole-config"
          mountPath: "/etc/goblackhole/config.yaml"
          subPath: config.yaml
      volumes:
        {{- with $.Values.deployment.volumes }}
          {{- toYaml . | nindent 8 }}
        {{- end }}
        - name: goblackhole-config
          configMap:
            name: goblackhole-config
```

Please test locally before accepting the changes, not sure if the indents are correct.